ledfx/api/log.py

```
@@ -11,13 +13,102 @@
 class LogEndpoint(RestEndpoint):
     ENDPOINT_PATH = "/api/log"
 
     def __init__(self, ledfx):
         self.logwebsocket = LogWebsocket(ledfx, ledfx.logqueue)
 
+        # In-memory rate limit: {ip: last_post_time}
+        self._rate_limit = {}
+        self._rate_limit_lock = threading.Lock()
+
+    async def post(self, request: web.Request, body) -> web.Response:
+        MAX_LEN = 200
```

Ruff (0.14.5)

Local variable `MAX_LEN` is assigned to but never used

Remove assignment to unused variable `MAX_LEN`

(F841)